canonical/store-components
# refactor: migrate stories from mdx to csf

## Done

This PR migrate component stories from MDX to CSF, please check issue https://github.com/canonical/store-components/issues/107 for more details

- Migrated all stories 1:1 from MDX to CSF
- Added a default layout for Docs on `preview.js`
- Added some types so docs won't show `any` when rendering the argtypes

## QA

### QA steps

- Run the project
- Open components stories
- Check that the behaviors haven't changed
- Open the code
- Check that there is less code, less cognitive overload and it has improved

## Fixes

Fixes: #107  .
